backend/utils/sidebar.py

```diff
@@ -0,0 +1,183 @@
+from utils.room_utils import DB, DEFAULT_DM_IMG, get_org_rooms
+
+
+class Sidebar:
+    """Serves as a data source for the sidebar
+
+    Sidebar class helps makes faster connection to core,
+    sorts out data, and creates the sidebar data format
+    for the frontend
+    """
+
+    @classmethod
+    async def __get_room_members(
+        cls, member_id: str, room: dict, org_members: list
+    ) -> dict:
+        """gets the room members excluding the current user
+
+        Args:
+            member_id (str): member_id of the current user
+            room (dict): room object data
+            org_members (list): list of all members in the organization
+
+        Returns:
+            [dict]: key value pair of room members
+                    example: {'member_id': {
+                                'username': str
+                                'image_url': str
+                                'starred': bool
+                                'role': str}
+                            }
```

Comment:
```suggestion
                    example: {'6gsg999449s949s4': {
                                'username': 'o4codes'
                                'image_url': 'http://ww.abc.vgn/er.jpeg'
                                'starred': false
                                'role': 'member'}
                            }
```
In the example, we should provide some concrete values instead of types.

Reply:
fixed